mlrun/model_monitoring/writer.py

```diff
@@ -154,12 +155,41 @@ def _update_kv_db(self, event: _AppResultEvent) -> None:
     def _generate_kv_schema(self, endpoint_id: str):
         """Generate V3IO KV schema file which will be used by the model monitoring applications dashboard in Grafana."""
         fields = [
-            {"name": WriterEvent.APPLICATION_NAME, "type": "string", "nullable": False},
-            {"name": WriterEvent.SCHEDULE_TIME, "type": "string", "nullable": False},
-            {"name": WriterEvent.RESULT_NAME, "type": "string", "nullable": False},
-            {"name": WriterEvent.RESULT_KIND, "type": "double", "nullable": False},
-            {"name": WriterEvent.RESULT_VALUE, "type": "double", "nullable": False},
-            {"name": WriterEvent.RESULT_STATUS, "type": "double", "nullable": False},
+            {
+                "name": WriterEvent.APPLICATION_NAME,
+                "type": "string",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.START_PROCESSING_TIME,
+                "type": "string",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.END_PROCESSING_TIME,
+                "type": "string",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.RESULT_NAME,
+                "type": "string",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.RESULT_KIND,
+                "type": "double",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.RESULT_VALUE,
+                "type": "double",
+                "nullable": False,
+            },
+            {
+                "name": WriterEvent.RESULT_STATUS,
+                "type": "double",
+                "nullable": False,
+            },
```

Comment:
Could you please change double -> int in kind and status?
https://github.com/mlrun/mlrun/pull/4495/files#r1375888171

Reply:
fixed - https://github.com/mlrun/mlrun/pull/4624/commits/f31e00b3c580fcd6c49f4f0e558fcac938c3e1ca
BTW, according to v3io docs, we can only use one of the following types - `string`, `double`, and `long` (https://github.com/v3io/v3io-py/blob/b869b0eb85d9f81fbe83218bac7265de635a6e2c/v3io/dataplane/kv.py#L359C1-L359C58).
However, I tested the flow with the new `int` type and it went well. I suspect that V3IO doesn't really care about the provided type... anyway let's leave it right now with `int`.